packages/contracts/.solcover.js

```diff
@@ -0,0 +1,35 @@
+const path = require('path')
```

Comment:
General explainer for this file: `solcover` only gives us the option to skip specific contracts during the instrumentation phase (so we can't explicitly list out the contracts we *do* want). `solcover` also doesn't allow us to use globs. This file pulls out the `testfiles` argument from the coverage script inside `package.json` and automatically determines the corresponding contract. It then skips all contracts *except* the corresponding contract.

Reply:
Could consider putting this explainer in as a comment in the file